apps/admin/frontend/src/components/reporting/filter_editor.tsx

```diff
@@ -114,6 +134,18 @@ function generateOptionsForFilter(
           label: 'Absentee',
         },
       ]);
+    case 'scanner':
+      return unique(scannerBatches.map((sb) => sb.scannerId)).map(
+        (scannerId) => ({
+          value: scannerId,
+          label: scannerId,
+        })
+      );
+    case 'batch':
+      return scannerBatches.map((sb) => ({
+        value: sb.batchId,
+        label: `${sb.scannerId} • ${sb.batchId.slice(0, 8)}`,
```

Comment:
I think the batch ID alone is too disorienting and needs to be accompanied by a scanner ID for some recognizability. I wasn't sure what to use as a separator here, so used "•". "-" seemed a little unclear that it was a separator. Open to whatever.

Reply:
LGTM! Was easy to understand at first glance when I watched the video